kpaamcamcommands/analyze.py

```diff
@@ -0,0 +1,135 @@
+"""
+
+"""
+import pathlib
+import itertools
+
+import pandas
+from lingpy import LexStat, Alignments, csv2list
+from lingpy.convert.plot import plot_heatmap
+from lingpy.evaluate.acd import bcubes
+
+from lexibank_kpaamcam import Dataset
+
+
+def register(parser):
+    pass
+
+
+def run(args):
+    ds = Dataset()
+    cldf = ds.cldf_reader()
+
+    def outpath(fname):
+        return str(ds.dir / 'analyses' / fname)
+
+    D = {0: ['concept', 'doculect', 'value', 'tokens']}
+
+    for i, ((concept, doculect), forms) in enumerate(itertools.groupby(
+        sorted(cldf['FormTable'], key=lambda f: (f['Parameter_ID'], f['Language_ID'])),
+        lambda f: (f['Parameter_ID'], f['Language_ID']),
+    ), start=1):
+        for form in forms:
+            D[i] = [concept, doculect, form['Form'], form['Segments']]
+            break  # We only take the first form variant per concept/doculect pair.
+
+    lex = LexStat(D)
```

Comment:
We can instantiate a `LexStat` object from the data `dict` created from the CLDF data - no need for a temporary file here.